### DISCUSSION TITLE
after upgrading to version 1.1.16 port 8444 is closed

### DISCUSSION THREAD

**Discussion by Gates207** [Chia-Network/chia-blockchain#5796] [Support]
After the upgrade, my port 8444 is closed. Despite this, synchronization is proceeding normally. Has anyone encountered such a problem? Perhaps this is a feature of the new update?
I did not make any changes to the computer settings. Before version 1.1.15, the port was open

I didn't make any changes to windows 10 settings. Ports are added to exceptions and UPNP is enabled in router settings. I also don’t have an antivirus. What could be the problem? Maybe there are some thoughts ...

**Comment by BrandtH22** [comment]
@Gates207 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
